ProjectSettings/EditorBuildSettings.asset

```diff
@@ -4,4 +4,8 @@
 EditorBuildSettings:
   m_ObjectHideFlags: 0
   serializedVersion: 2
-  m_Scenes: []
+  m_Scenes:
```

Comment:
Please revert this file. 
You are attempting to check in your Editor build settings to the entire project.